result/accessor-pairs.js

```diff
@@ -0,0 +1,99 @@
+// Bad
+var o = {
+    set a(value) {
+        this.val = value;
+    }
+};
+
+// Good
+var o = {
+    set a(value) {
+        this.val = value;
+    },
+    get a() {
+        return this.val;
+    }
+};
+
+/*eslint accessor-pairs: "error"*/
+
+var o = {
```

Comment:
__[development][eslint]__
* Error - 'o' is already defined (no-redeclare)
* Error - Getter is not present (accessor-pairs)